### ISSUE TITLE
"Missing keys" warning

### ISSUE THREAD

**Issue by Tobbe** [onfido/onfido-sdk-ui#1023]
### What was the expected behaviour?

No "Missing keys" warning in console

### What happened instead?

![image](https://user-images.githubusercontent.com/30793/83410070-f6329500-a415-11ea-8623-7944c38293ec.png)

Even though I get a warning about missing keys, it still seems to use my mobilePhrases keys anyway, so it's a false warning.

### Version info:

- Onfido SDK 5.9.2
- OS Windows 10
- Browser Chrome
- Device Desktop
- Node N/A
- Framework or relevant dependencies (if applicable) N/A

### Integration configuration:

```
init({
  useModal: false,
  steps: ["document", "face", "complete"],
  token,
  containerId: "onfido-mount",
  language: {
    locale: "da_DK",
    phrases: getPhrases(intl),
    mobilePhrases: getMobilePhrases(intl),
  },
  onComplete: (data: any) => {
    console.log("everything is complete", JSON.stringify(data));
  },
});
```

If I instead do this, the warning goes away

```
init({
  useModal: false,
  steps: ["document", "face", "complete"],
  token,
  containerId: "onfido-mount",
  language: {
    locale: "da_DK",
    phrases: {
      ...getPhrases(intl),
      mobilePhrases: getMobilePhrases(intl),
    },
    mobilePhrases: getMobilePhrases(intl),
  },
  onComplete: (data: any) => {
    console.log("everything is complete", JSON.stringify(data));
    confirm();
    setIsDone(true);
  },
});
```

But that's just redundant, and shouldn't be needed

### Steps to reproduce:

Configure custom translations as above, and watch your console for warning messages

### Demo:

Sorry, don't have anything public to show at the moment

**Comment by stefaniacardenas** [comment]
Hi @Tobbe, thanks for reporting this issue. I was able to reproduce it and I will add it to our backlog. I will post an update here when a fix goes live.

**Comment by stefaniacardenas** [comment]
Hi, this issue has been fixed in version [5.11.0](https://github.com/onfido/onfido-sdk-ui/releases/tag/5.11.0).
